### PR TITLE
Bugfix when standardizing scss and responsive page

### PR DESCRIPTION
**Pull request by DrielisonLopes**
- Bug fixes after scss standardizing fixed
- Refactoring in About Games to facilitate mobile version
- Responsiveness on the page
- Creation scss variables for use media queries
- alias for variables.scss in tsconfig
- Remove styles from html tags
- Delete useless global.module

### CODE REVIEW

File: src/components/atoms/Footer.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React from "react";
 export default function Footer() {
   return (
     <div style={{ textAlignLast: "center" }}>
-      <p>Pixels Forge Studio - Copyright © 2023</p>
+      <p style={{ paddingBottom: "16px" }}>Pixels Forge Studio - Copyright © 2023</p>
```
Comment (renanlopescoder) [inline src/components/atoms/Footer.tsx:6]: Should we use `Footer.module.scss` here?

File: src/components/molecules/AboutGame/AboutGame.module.scss [hunk 2]
```diff
@@ -1,15 +1,24 @@
+@import "@/variables";
```
Comment (renanlopescoder) [inline src/components/molecules/AboutGame/AboutGame.module.scss:1]: Niceee 👍🏻

**Review by renanlopescoder — CHANGES_REQUESTED** [review]
Just need one update on the Footer component

File: src/components/atoms/Footer.tsx (hunk 1 above)
Comment (DrielisonLopes) [reply src/components/atoms/Footer.tsx:6]: I ended up not doing this one at the moment because it was very simple but if I improve the footer later yes :)

**Review by renanlopescoder — APPROVED** [review]

**Pull request merged**